storage/README.md

```diff
@@ -1,8 +1,8 @@
-## Cloud Storage [![Go Reference](https://pkg.go.dev/badge/cloud.google.com/go/storage.svg)](https://pkg.go.dev/cloud.google.com/go/storage)
+## Cloud Storage [![Go Reference](https://pkg.go.dev/badge/cloud.google.com/go/storage.svg)]([https://pkg.go.dev/cloud.google.com/go/storage](https://cloud.google.com/go/docs/reference/cloud.google.com/go/storage/latest))
```

Comment:
Not sure what to do with this badge....

Reply:
I feel like the badge should stick with pkg.go.dev, since it's standard across Go packages.

Reply:
Agreed. Switched it back